dataPipelines/gc_thumbnails/utils.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+import fitz
+import typing as t
+
+class ThumbnailsCreator:
+    def __init__(
+        self,
+        input_directory: t.Union[str, Path],
+        output_directory: t.Union[str, Path]
+    ):
+        self.input_directory = Path(input_directory).absolute()
+        self.output_directory = Path(output_directory).absolute()
+
+    def generate_thumbnails(self):
+        print('\nGenerating Thumbnails\n')
+        self.output_directory.mkdir(exist_ok=True)
+        for file_path in [x for x in self.input_directory.glob('*.pdf')]:
```

Comment:
you can reduce it to `for file_path in self.input_directory.glob('*.pdf'):` , since `Path.glob()` returns an iterable suitable for the for loop.